README.md

```diff
@@ -61,35 +54,60 @@ sudo snap install kubectl --classic
 sudo snap install helm --classic
 ```
 
-### Local Environment Setup
-
-- Create a virtual environment `venv` using `python3 -m venv venv`
-- Upgrade `pip3` using `pip3 install pip --upgrade` to ensure you are using the latest version of pip.
-- Activate `venv` using `. venv/bin/activate`
-- Install the python dependencies: `pip3 install -r requirements.txt`
-- If you get an error about the version of setuptools, upgrade it manually using `pip3 install setuptools --upgrade`
-- Clone this repository and `cd` into it
-- Install Ansible requirements `ansible-galaxy collection install -r requirements.yml`
-
-## GPU
-
-- GPU drivers will automatically be pulled from the Nvidia NGC catalogue
-
-## Kubectl Namespaces
-
-All components are installed in the user-selected namespace and the Prometheus-stack is deployed to the `prometheus` namespace. This means every command must:
-
-- Include `-n <namespace>` on every command
-- (Preferred) use [Kubectx and kubens](https://github.com/ahmetb/kubectx) and set the namespace for this session with `kubectl ns jhub`
-
-All subsequent Kubernetes commands will omit the namespace for brevity.
+## Deploying JupyterHub
 
-## JupyterHub Config
+### Instructions
 
-- Ensure that the terminal Ansible will run in can access the correct cluster (`kubectl get no`).
-- Check the config in `playbooks/deploy_jhub.yml`.
-- Copy the content of `config.yaml.template` (in `/roles/deploy_jhub/files/`) to create a `config.yaml` in the same directory and ensure the various secrets and fields marked: `<Not Set>` are completed.
-- Go through each line checking the config values are as expected. Additional guidance is provided below:
+1. Deploy a capi cluster
+2. Ensure that you can access the cluster from the machine you are running this playbook from (`kubectl get no`)
+3. git clone this repo (`git clone https://github.com/stfc/ansible-jupyter`)
+4. Setup virtual environment:\
+  a. Create a virtual environment `venv` using `python3 -m venv venv`\
+  b. Upgrade `pip3` using `pip3 install pip --upgrade` to ensure you are using the latest version of pip.\
+  c. Activate `venv` using `. venv/bin/activate`\
+  d. Install the python dependencies: `pip3 install -r requirements.txt`\
+  e. If you get an error about the version of setuptools, upgrade it manually using `pip3 install setuptools --upgrade`\
+  f. Install Ansible requirements `ansible-galaxy collection install -r requirements.yml`\
+5. Uncomment the correct line for your environment in `inventory/hosts`
+6. Fill in the variables for your given environment in `group_vars/<environment>/all.yaml`
+    - `admin_names`: The admin usernames to be created (these will be prepended with `admin-`)
+    - `number_of_users`: The number of user accounts to be created
+    - `staginger_cert`: whether to use acme to generate a staging cert
+    - `nfs_ip`: The IP address of the nfs server
+    $\\[5mm]$ </p>
```

Comment:
This line is visible in the generated markdown (and for other lines further down). I think the `$\\[5mm]$` lines could be removed from the file?